libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -990,6 +991,48 @@ impl NodeAPI for Greenlight {
         debug!("send_custom_message returned status {:?}", resp.status);
         Ok(())
     }
+
+    #[allow(unused_variables)]
+    async fn fetch_invoice(
+        &self,
+        offer: String,
+        amount_msat: Option<u64>,
+        quantity: Option<u64>,
+        recurrence_counter: Option<u64>,
+        recurrence_start: Option<f64>,
+        recurrence_label: Option<String>,
```

Comment:
Maybe it makes sense to remove the recurrence options until there is a clear plan on how recurrence can fit into the SDK. Recurrence has to be programmed on the client side, so inside the SDK or the calling application somehow. I think it makes sense to explicitly enable that once we know how to handle it. Unless you know how to handle it!